qa/pull-tester/rpc-tests.py

```diff
@@ -37,7 +38,7 @@
     # terminal via ANSI escape sequences:
     BOLD = ('\033[0m', '\033[1m')
 
-RPC_TESTS_DIR = BUILDDIR + '/qa/rpc-tests/'
+RPC_TESTS_DIR = SRCDIR + '/qa/rpc-tests/'
```

Comment:
Sure, this is on my list of cleanups. If no one beats me to it, you can expect it some time after feature freeze. ;)